From 983d122f5af6de6b8112cba244a4d566ac11b733 Mon Sep 17 00:00:00 2001
From: Robert Kanter <rkanter@apache.org>
Date: Thu, 6 Apr 2017 17:44:47 -0700
Subject: [PATCH 2406/2748] MAPREDUCE-6201. TestNetworkedJob fails on trunk
 (pbacsko via rkanter)

(cherry picked from commit e7167e4a1323cff3edfa45750b927b198c5fbc0a)

Change-Id: Ie69bd0881399b9c9810f79117635be9364f66c76
---
 .../org/apache/hadoop/mapred/TestNetworkedJob.java |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/TestNetworkedJob.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/TestNetworkedJob.java
index 00cf4f9..fa1496b 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/TestNetworkedJob.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/TestNetworkedJob.java
@@ -397,6 +397,9 @@ private MiniMRClientCluster createMiniClusterWithCapacityScheduler()
     // Expected queue names depending on Capacity Scheduler queue naming
     conf.setClass(YarnConfiguration.RM_SCHEDULER, CapacityScheduler.class,
         CapacityScheduler.class);
+    // Default value is 90 - if you have low disk space,
+    // testNetworkedJob will fail
+    conf.set(YarnConfiguration.NM_MAX_PER_DISK_UTILIZATION_PERCENTAGE, "99");
     return MiniMRClientClusterFactory.create(this.getClass(), 2, conf);
   }
 }
-- 
1.7.9.5

